scripts/build/TestPlatform.Settings.targets

```diff
@@ -2,11 +2,12 @@
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <TestPlatformRoot Condition="$(TestPlatformRoot) == ''">$(MSBuildThisFileDirectory)../../</TestPlatformRoot>
+    <TPVersionPrefix>15.7.0</TPVersionPrefix>
```

Comment:
Shouldn't this be 15.6.0, we haven't RTM yet

Reply:
Created rel/15.6 branch.